tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/project/ProjectNameValidationTest.java

```diff
@@ -38,61 +43,51 @@
 	public static final String PROJECT_NAME_MAX_LENGTH_ERROR = 
 			" Maximum length allowed is 63 characters for project name";
 	
+	@Before
+	public void setUp() {
+		openNewProjectShell();
+	}
+	
+	@After
+	public void cleanUp() {
+		closeNewProjectShell();
+	}
+	
 	@Test
 	public void testShortProjectName() {
-		openNewProjectShell();
 		new LabeledText(OpenShiftLabel.TextLabels.PROJECT_NAME).setText("s");
-		
-		try {
-			new DefaultText(PROJECT_NAME_SHORT_ERROR);
-		} catch (RedDeerException ex) {
-			fail("There is no validation message warning about small length of a project name.");
-		}
-		
-		closeNewProjectShell();
+		new DefaultText(PROJECT_NAME_SHORT_ERROR);
```

Comment:
I do get arguments of both sides.  It's kind of frog-mice war right now. I agree with Marian that the crucial part of each test should be done by some assertion (or as he is doing it, because we don't have better mechanism right now). 
But even despite that, I'm for merging this PR as it is. It hugely increases readability and it's possible (quite easily) to see from the exception where the error occurs.
We'll think of how to implement assertions into RedDeer framework so we can do this properly next time ;-)
I talked to Marian about my opinion and he agreed that this PR can be merged (but he is still not happy about that :-D).
If Jeff and Alexey want to provide opinion it's still desired. The more valid points are made, the better code we'll have.